protocols/request-response/src/cbor.rs

```diff
@@ -44,11 +45,18 @@ pub type Behaviour<Req, Resp> = crate::Behaviour<codec::Codec<Req, Resp>>;
 
 mod codec {
     use async_trait::async_trait;
+    use cbor4ii::{core::error as error_cbor, serde as serde_cbor};
```

Comment:
I'd prefer to not rename imports and instead just reference the crate `cbor4ii` directly.